content/applications/sales/sales/gelato.rst

```diff
@@ -1,7 +1,167 @@
+get the product ID in Gelato, navigate to the :guilabel:`Templates` page from the side bar menu.
+Select which product to synchronize in Odoo, then hover over the product card to reveal the
+:icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis)` menu icon. Click the menu icon, then click
+:guilabel:`Copy Template ID` to copy the product template ID to the clipboard.
+
+.. seealso::
+   `Start selling products with Gelato: Quick & easy setup
+   <https://www.gelato.com/blog/get-started-with-gelato-creating-products>`_
+
+
+Odoo Sales product
+------------------
+
+To create a product in Odoo that matches the Gelato one, navigate to :menuselection:`Sales app -->
+Products --> Products`, select :guilabel:`New` to create a new product form. Type in the product
+:guilabel:`Name`, then navigate to the :guilabel:`Sales` tab. Find the :guilabel:`Gelato` section,
+then click into the :guilabel:`Template Reference` field and paste the copied template ID from the
+Gelato product. Finally, click :guilabel:`Synchronize`.
+
+Successful synchronization pulls the Gelato product variant options into the newly configured Odoo
+product.
+
+In the new :guilabel:`Print Images` field, click the :guilabel:`default` marker to set a default
+product image. Click the :icon:`fa-pencil` :guilabel:`(edit)` icon and select the product image file
+to upload, then :guilabel:`Save & Close`.
+
+.. important::
+   The :guilabel:`Print Images` field *must* be configured on the product before it can be ordered.
+
+Product variants
+----------------
+
+To view and edit the newly synchronized product variants, navigate to the :guilabel:`Attributes &
+Variants` tab, which will have the variants pulled from the Gelato product configuration. Click the
+:guilabel:`Configure` button to edit and configure the variant images, delivery methods, additional
+pricing, etc.
+
+Order a Gelato product from Odoo
+--------------------------------
+
+Once synchronized, Gelato products are available to order in Odoo through :doc:`sales quotations
+<send_quotations>` or on the **eCommerce** store. Gelato delivery options are automatically
+synchronized upon API and webhook configuration.
+
+To add Gelato delivery, click :guilabel:`Add shipping` on the sales order. Select
+:guilabel:`Standard Delivery` or :guilabel:`Express Delivery` in the :guilabel:`Shipping Method`
+field, then click :guilabel:`Get rate`.
+
+Once the quotation is confirmed, it becomes an active sales order, and the order is sent to Gelato
+for fulfillment. Once a sales order is sent from Odoo to Gelato, Gelato processes the order,
+produces the product at the nearest fulfillment center, and ships it directly to the end-customer.
+
+.. seealso::
+   :doc:`send_quotations/create_quotations`
+
+.. important::
+   When creating a sales order for Gelato products in the database, only Gelato products can be
+   added to the same sales order. Multivendor orders are not available with the Gelato connector at
+   this time.
```

Comment:
I don't have a strong opinion on this but I'm not sure if it's worth mentioning a behavior that is enforced and made pretty clear already in Odoo.